index.md

```diff
@@ -536,6 +563,21 @@ You can supplement the poster by:
 	- handing out visiting cards to interested people, or
 	- my favourite option &mdash; providing slips pointing to a website where you host more material/information, such as videos, photographs, papers, reports etc. (include this URL on the poster as well).
 
+## Linux
+
+(… or more correctly, since I'm not speaking exclusively of the kernel, "the GNU/Linux operating system and its various distributions".
+But that's [a conversation for another day](https://en.wikipedia.org/wiki/GNU/Linux_naming_controversy), and for the purposes of this section I'm going to use "Linux" as a shorthand term.)
+
+I don't use Linux for my academic work, but I do have [Linux distros](https://en.wikipedia.org/wiki/Linux_distribution) running on three machines at home and work.
+Unless you already have a MacBook running macOS (a UNIX OS, allowing you to relatively trivially set up the development tools you need) or are required to use Windows by your university in order to access various in-house tools, you might want to give Linux a spin.
+Not only do many of the aforementioned software tools and packages run natively and nearly flawlessly on it, Linux itself is also free as in free beer (in addition to respecting your freedoms).
+
+### The terminal
+
+(This section isn't strictly about Linux -- I use the terminal on macOS, myself -- but it makes some sense to include it here.)
+Personally, learning to use the terminal ([Bash](https://en.wikipedia.org/wiki/Bash_(Unix_shell)) and the [command-line interface](https://en.wikipedia.org/wiki/Command-line_interface) in general) has been a huge help to me with my work.
+If you're looking for a good hands-on tutorial, check out Rik Smith-Unna's [command-line bootcamp](http://rik.smith-unna.com/command_line_bootcamp/).
+
 ## Support Structures
```

Comment:
There's a few more Twitter communities, e.g. @PhDpositivity, Shut up and Write, @VivaSurvivors, @GradHacker, @HackyourPhD, @Dissertating, #phdlife...